### ISSUE TITLE
upgrade to latest markdown version

### ISSUE THREAD

**Issue by ammaratef45** [flutter/flutter_markdown#293]
I get the following error when I add markdown `3.0.0` with flutter markdown `0.4.4`

```
Running "flutter pub get" in flutter_markdown_editor...         
Because flutter_markdown 0.4.4 depends on markdown ^2.1.3 and no versions of flutter_markdown match >0.4.4 <0.5.0, flutter_markdown ^0.4.4 requires markdown ^2.1.3.
So, because flutter_markdown_editor depends on both flutter_markdown ^0.4.4 and markdown ^3.0.0, version solving failed.

pub get failed (1; So, because flutter_markdown_editor depends on both flutter_markdown ^0.4.4 and markdown ^3.0.0, version solving failed.)
exit code 1

**Comment by domesticmouse** [comment]
We will be releasing `flutter_markdown` 0.5.0 which uses `markdown` 3.0.0 shortly.

**Comment by ammaratef45** [comment]
@domesticmouse That sounds great, thanks

**Comment by domesticmouse** [comment]
The fix for this has been released in v0.5.0. Thanks!
